static/purchase.js

```diff
@@ -0,0 +1,63 @@
+// ----- Setup ----- //
+
+const DRIVER_TOTAL = 1000;
+const TEAM_TOTAL = 1000;
+
+
+// ----- Functions ----- //
+
+// Retrieves the drivers and teams from the page.
+function getElems () {
+
+    return {
+	drivers: document.querySelectorAll('.driver-table input[type=checkbox]:checked'),
+	teams: document.querySelectorAll('.team-table input[type=checkbox]:checked')
+    };
+    
+}
+
+// Sums a list of objects with a price field.
+function sumPrices (total, {price}) {
+    return total + parseInt(price);
+}
+
+// Calculates whether the spend is within budget.
+function withinBudget (drivers, teams) {
+
+    let driverTotal = drivers.reduce(sumPrices, 0);
+    let teamTotal = teams.reduce(sumPrices, 0);
+    console.log(driverTotal, teamTotal);
```

Comment:
The console log? That's there in lieu of anything useful actually appearing on the page =). That's the only output atm I think.